app/src/main/java/com/github/se/orator/ui/settings/SettingsScreen.kt

```diff
@@ -165,15 +120,27 @@ fun SettingsScreen(
                     titleContentColor = MaterialTheme.colorScheme.onSurface,
                     actionIconContentColor = MaterialTheme.colorScheme.onSurfaceVariant,
                 ))
-      },
-      content = { padding ->
-        Column(
+      }) { padding ->
+        LazyColumn(
             modifier = Modifier.fillMaxSize().padding(padding).testTag("settingsScreen"),
             verticalArrangement = Arrangement.spacedBy(AppDimensions.spacerWidthMedium)) {
-              for (setting in listOfSettings) {
-                TextButtonFun(setting, themeViewModel)
-                HorizontalDivider(thickness = AppDimensions.dividerThickness)
+
+              // Permissions
+              item {
+                TextButtonFun(listOfSettings[0]) {
+                  context.startActivity(
+                      Intent(Settings.ACTION_APPLICATION_DETAILS_SETTINGS).apply {
+                        data = Uri.fromParts("package", context.packageName, null)
+                      })
+                }
               }
+              item {
```

Comment:
Since you added a comment // Permissions I would suggest adding another one like // Switch theme

Reply:
My permission comment was to make it clear that the code redirected to the settings of the device. I don't think "symmetry" of the comments is really a good thing honestly, they should just be added where the code could be hard to understand.